### PR TITLE
RTP15c

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RealtimeClientPresence.swift [hunk 1]
```diff
@@ -1283,6 +1283,32 @@ class RealtimeClientPresence: QuickSpec {
...
+                        channel.presence.updateClient("john", data:"mobile") { error in
+                            expect(error).to(beNil())
+                            done()
+                        }
+                    }
+                }
+
+                waitUntil(timeout: testTimeout) { done in
+                    channel.presence.get() { members, error in
+                        expect(members!.first!.data as? String).to(equal("mobile"))
```
Comment (mattheworiordan) [inline Spec/RealtimeClientPresence.swift]: Why are you expecting `mobile` first, ordering of the member set is not guaranteed.  Also, the tests says that you you should make sure it has no side effects, so you should check that both members exist

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: But the test has only one member. I will change it to use two clientId's to be more complete.

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: Done 08a0a3e.

**Comment by tcard** [comment]
LGTM

**Pull request merged**